test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -135,7 +135,7 @@ u256 EwasmBuiltinInterpreter::evalBuiltin(
 			return u256(util::keccak256(arg)) & 0xfff;
 		else if (fun == "dataoffset")
 		{
-			arg[31] += 2;
+			arg[31] += static_cast<char>(2);
```

Comment:
Some compilers freaked out from `char(2)` or even just `2`...

Reply:
yes, because the signedness of `char` is kind of unspecified...